src/SelectTrigger.jsx

```diff
@@ -102,7 +108,26 @@ export default class SelectTrigger extends React.Component {
         dropdownMenuStyle={props.dropdownMenuStyle}
         menuItemSelectedIcon={props.menuItemSelectedIcon}
       />
-    );
+    )
+
+    if (dropdownRender) {
+      return (
+        <div
```

Comment:
Can we remove this div? Let's return the same content if user call with empty render:
```jsx
dropdownRender={menu => menu}
```

Reply:
The global of this div is preventing blur event when clicking the menu inside. There is a race condition between click and blur event outside such as antd select onBlur, it can be removed technically, thus the developer takes charge of this, but it will be a pretty annoy case because dev should take it into consideration in their project.
I agree to the remaining same point and be willing to hear your advice about this.

Reply:
This should be enhanced by `rc-trigger` to handle blur logic. Maybe should in another PR for the trigger.